src/Data/Food/Retail.elm

```diff
@@ -0,0 +1,193 @@
+module Data.Food.Retail exposing (Conservation, all, ambient, computeImpacts, decode, displayNeeds, encode, fromString, toDisplay, toString)
+
+{- This module allow to compute the impacts of the transport of finished products to the retail stores,
+   and the impact of storing the product at the store
+-}
+
+import Data.Food.Builder.Db exposing (Db)
+import Data.Food.Process exposing (Process, WellKnown)
+import Data.Impact as Impact exposing (Impacts)
+import Data.Unit as Unit
+import Energy exposing (Joules, kilowattHours)
+import Json.Decode as Decode exposing (Decoder)
+import Json.Encode as Encode
+import Length exposing (Length)
+import Mass exposing (Mass)
+import Quantity exposing (Quantity, Rate, rate, ratio)
+import Result.Extra as RE
+import Volume exposing (CubicMeters, Volume, cubicMeters, liters)
+
+
+type
+    Conservation
+    -- A consevation type and its needs in energy, cooling, water and transport
+    = Conservation Type Needs
+
+
+type Type
+    = Ambient
+    | Chilled
+    | Frozen
```

Comment:
How about using a slightly more useful type name, like `Refrigeration`? 🤔

Reply:
Replaced Chilled with Fresh, which is shorter and consistent with other adjectives